### DISCUSSION TITLE
Modificar header Accept sin interceptor

### DISCUSSION THREAD

**Discussion by WilianMarroquin** [manchenkoff/nuxt-auth-sanctum#276] [Q&A] [answered]
"¿Es posible modificar el encabezado Accept: application/json directamente en las solicitudes sin necesidad de utilizar un interceptor? En caso afirmativo, ¿podrían proporcionar un ejemplo de cómo configurarlo correctamente?"

**Comment by manchenkoff** [comment, marked as answer]
Hey @WilianMarroquin, since this module provides a simple wrapper on top of `ofetch`, you can try doing something like this

```typescript
const sanctumFetch = useSanctumClient()

await sanctumFetch(
  '/api/endpoint',
  {
    headers: {
      CustomHeader: 'custom-value',
    },
  },
)
```

However, the module may override some of the values because the header processing order might differ. Another disadvantage is that you have to repeat the same approach every time you want to pass a custom header, so interceptors can solve this more efficiently.

**Reply by WilianMarroquin** [reply]
¡Gracias por el apoyo, considerare usar interceptores! 🙌
